style.css

```diff
@@ -0,0 +1,95 @@
+/* @media all and (max-width: 768px) { */
+* {
+  font-family: sans-serif;
+  margin: 0;
+  padding: 0%;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100vw;
+  height: auto;
+}
+
+.coverpage {
+  width: 100vw;
+  height: auto;
+  margin-bottom: 6rem;
+  padding: 1%;
+  background-color: white;
+  border-bottom-left-radius: 80px;
+}
```

Comment:
- I believe it will be better if we fixed all margins and padding in the mobile portfolio. We can start by adding some extra space to the left and right sides so that it looks like Figma to a good extent, then the spaces above and below the paragraph should be reduced. 🤗 

Here is what I mean 👇🏽 
| Your Design  |Figma Design |
| :---:   | :---: | 
| <img width="385" alt="Screenshot 2023-01-12 at 00 20 14" src="https://user-images.githubusercontent.com/99246359/211993091-324e87ac-cf05-449d-91a2-11babe5d5566.png"> |<img width="374" alt="Screenshot 2023-01-12 at 00 19 39" src="https://user-images.githubusercontent.com/99246359/211993170-aa5074df-4f95-49fc-9b9d-20bdc848f17c.png">  |

Reply:
@Favourezeugwa would you please update my submission status so that I can make the changes

Reply:
This is great work you've done on the UI 🚀 but the reviewer's requested changes haven't been addressed, and the design still lacks padding and margin. See below 👇🏽

![Screenshot 2023-01-12 at 23 21 46](https://user-images.githubusercontent.com/45739963/212236767-c7d0ca82-16fa-4d02-ad99-f47f58cd6e90.png)